src/App.js

```diff
@@ -1,33 +1,35 @@
 import React, { Component } from 'react';
-import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { MovieFilter } from './components/MovieFilter';
+import './App.scss';
 
 export class App extends Component {
-  state = {};
+  state = {
+    movies: moviesFromServer,
+  }
+
+  queryChangeHandler = (query) => {
+    const queryLowerCase = query.toLowerCase();
+
+    this.setState({
+      movies: moviesFromServer.filter(({ title, description }) => (
+        title.toLowerCase().includes(queryLowerCase)
+          || description.toLowerCase().includes(queryLowerCase)
+      )),
+    });
```

Comment:
actually where's no need to save movies to the state and then update them, we can filter moviesFromServer
let's create a variable filteredMovies and move this code there, then we can pass them directly to the MovieList like this
`<MoviesList movies={filteredMovies} />`